common/app/googleAuth/FilterExemptions.scala

```diff
@@ -1,17 +1,15 @@
-package http
+package googleAuth
 
 import com.gu.googleauth.FilterExemption
 
-object FilterExemptions {
+case class FilterExemptions(additionalUrls: String*) {
 
   lazy val loginExemption: FilterExemption = FilterExemption("/login")
   lazy val exemptions: Seq[FilterExemption] = List(
+    // Default
     FilterExemption("/oauth2callback"),
     FilterExemption("/assets"),
     FilterExemption("/favicon.ico"),
-    FilterExemption("/_healthcheck"),
-    FilterExemption("/2015-06-24-manifest.json"),
-    // the healthcheck url
-    FilterExemption("/world/2012/sep/11/barcelona-march-catalan-independence")
```

Comment:
Thanks for explaining @TBonnin ! All the healthcheck urls are being added here: https://github.com/guardian/frontend/blob/0295a1225ce48984dc648c6c590bec502f399717/preview/app/http/PreviewFilters.scala#L17